tsdb/db.go

```diff
@@ -880,7 +884,11 @@ func (db *DB) reload() (err error) {
 				block.Close()
 			}
 		}
-		return fmt.Errorf("unexpected corrupted block:%v", corrupted)
+		var merr tsdb_errors.MultiError
+		for ulid, err := range corrupted {
+			merr.Add(errors.Wrap(err, ulid.String()))
+		}
+		return merr.Err()
```

Comment:
I think we lost wrapping here.. should we had errors.Wrap here as well?

Reply:
done